src/Libraries/CoreNodes/String.cs

```diff
@@ -310,15 +310,22 @@ public static int IndexOf(string str, string searchFor, bool ignoreCase = false)
                     ? StringComparison.InvariantCultureIgnoreCase
                     : StringComparison.InvariantCulture);
         }
-
-        public static int[] AllIndicesOf(string str, string searchFor, bool ignoreCase = false)
+        /// <summary>
+        /// Finds list of indices where sub-string appears inside a string.
+        /// </summary>
+        /// <param name="string">A string to search in.</param>
+        /// <param name="searchFor">Substring to search for.</param>
+        /// <param name="ignoreCase">Whether or not comparison takes case into account.</param>
+        /// <returns name="indexList">List of indices (integers) where substring is found</returns>
+        /// /// <search>all indices of,find substring,where,search</search>
```

Comment:
looks like there are some extra `///` here

Reply:
addressed in 
64fe05b